src/date-range-picker/interfaces.ts

```diff
@@ -91,8 +91,7 @@ export interface DateRangePickerBaseProps {
    *
    * Default: the user's current time offset as provided by the browser.
    *
-   * This property is deprecated. Use getTimeOffset instead.
-   * @deprecated
+   * @deprecated Use getTimeOffset instead.
```

Comment:
Since we're going for consistency, can we also make sure that we are properly formatting property names as inline code? (i.e. `getTimeOffset`)

Reply:
Good point! Updated 👍